packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.routes.js

```diff
@@ -2,11 +2,15 @@ import template from './ip-ip-agoraOrder.html';
 import controller from './ip-ip-agoraOrder.controller';
 
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('app.ip.dashboard.agora-order', {
+  $stateProvider.state('app.ip.agora-order', {
     url: '/agoraOrder?{catalogName:string}',
-    template,
-    controller,
-    controllerAs: '$ctrl',
+    views: {
+      ipview: {
+        template,
+        controller,
+        controllerAs: '$ctrl',
+      },
```

Comment:
why not using component ?

Reply:
HI Mohammed, Yes we can do it. Can we do it as part of refactor ?

Reply:
Yep, we can you have to create Jira ticket to not forget it